Pipfile

```diff
@@ -13,6 +13,7 @@ requests = "~=2.31"
 psutil = "~=5.9"
 celery = "~=5.3.5"  # Celery v5.4 may stop supporting Python 3.8
 django-celery-results = "~=2.5"
+django-markdownify = "~=0.9.3"  # includes the markdown package
```

Comment:
Thoughts on only fixing the major version? (Using `~=0.9` instead)

Reply:
Just took a look at the changelog, seems fine to me.